staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -285,26 +288,30 @@ func (s preparedGenericAPIServer) Run(stopCh <-chan struct{}) error {
 func (s preparedGenericAPIServer) NonBlockingRun(stopCh <-chan struct{}) error {
 	// Use an internal stop channel to allow cleanup of the listeners on error.
 	internalStopCh := make(chan struct{})
-
```

Comment:
would leave this empty line. These two blocks are not related.

Reply:
ok